src/typography/src/Text.tsx

```diff
@@ -10,17 +10,17 @@ export interface TextProps extends React.ComponentProps<typeof Box> {
   /**
    * The color (alias or valid color) applied to the text
    */
-  color?: string
+  color: string
 
   /**
    * The font family alias applied to the text
    */
-  fontFamily?: FontFamily
```

Comment:
These aren't really required though – are they?

Reply:
They are, but they have `defaultProps` so you don't need to set them.